app/views/WhatsRunningWherePage.scala.html

```diff
@@ -58,7 +58,7 @@ <h1>What's Running Where?</h1>
             <tbody class="list">
             @whatsRunning.zipWithIndex.map { case (wrw, i) =>
             <tr role="row" id="row@{i}_name">
-                <td class="application-name">@{wrw.applicationName.asString}</td>
+                <td class="application-name"><a href="/service/@{wrw.applicationName.asString}">@{wrw.applicationName.asString}</a></td>
```

Comment:
Can you use reverse routes, rather than hard-coding the url (it will then be validated in compile time)

Reply:
Yes, my bad. Have pushed again with a resolution to this.